pkg/reconciler/workload/synctarget/synctarget_reconcile.go

```diff
@@ -30,7 +30,7 @@ import (
 	virtualworkspacesoptions "github.com/kcp-dev/kcp/cmd/virtual-workspaces/options"
 	corev1alpha1 "github.com/kcp-dev/kcp/pkg/apis/core/v1alpha1"
 	workloadv1alpha1 "github.com/kcp-dev/kcp/pkg/apis/workload/v1alpha1"
-	syncerbuilder "github.com/kcp-dev/kcp/pkg/virtual/syncer/builder"
+	syncerbuilder "github.com/kcp-dev/kcp/tmc/pkg/virtual/syncer/builder"
```

Comment:
Does this mean that syncer is now part of the TMC stack? Moves out of the core with this change?

Reply:
the virtual workspace for syncer, yes